Websites/perf.webkit.org/tools/js/buildbot-triggerable.js

```diff
@@ -321,7 +332,7 @@ class BuildbotTriggerable {
         for (let request of buildRequests) {
             let groupId = request.testGroupId();
             if (!map.has(groupId)) // Don't use real TestGroup objects to avoid executing postgres query in the server
-                map.set(groupId, {id: groupId, groupOrder: groupOrder++, requests: [request], buildSyncer: null, testSyncer: null, workerName: null});
```

Comment:
Is this a bug fix some sort? It would be good to mention in the commit message.

Reply:
Now a test group can have multiple build syncers, and we no longer need this to tie a builder to a test group.
I mentioned `Make build type requests can be scheduled in parallel by removing the assumption that a test group only have one build syncer.`, I will also include this information in the summary of the commit message.

Reply:
Done